test/powershell/Host/TabCompletion/TabCompletion.Tests.ps1

```diff
@@ -1,97 +1,104 @@
-    } 
+    }
 
     It 'Should complete format-wide hashtable' {
-        $res = TabExpansion2 -inputScript 'Get-ChildItem | Format-Wide @{ ' -cursorColumn 'Get-ChildItem | Format-Wide @{ '.Length 
+        $res = TabExpansion2 -inputScript 'Get-ChildItem | Format-Wide @{ ' -cursorColumn 'Get-ChildItem | Format-Wide @{ '.Length
         $res.CompletionMatches.Count | Should Be 2
         $res.CompletionMatches.Foreach{$_.CompletionText -in 'Expression', 'FormatString' | Should Be $true}
-    } 
+    }
 
     It 'Should complete format-custom  hashtable' {
-        $res = TabExpansion2 -inputScript 'Get-ChildItem | Format-Custom @{ ' -cursorColumn 'Get-ChildItem | Format-Custom @{ '.Length 
+        $res = TabExpansion2 -inputScript 'Get-ChildItem | Format-Custom @{ ' -cursorColumn 'Get-ChildItem | Format-Custom @{ '.Length
         $res.CompletionMatches.Count | Should Be 2
         $res.CompletionMatches.Foreach{$_.CompletionText -in 'Expression', 'Depth' | Should Be $true}
-    } 
-    
+    }
+
     It 'Should complete Select-Object  hashtable' {
-        $res = TabExpansion2 -inputScript 'Get-ChildItem | Select-Object @{ ' -cursorColumn 'Get-ChildItem | Select-Object @{ '.Length 
+        $res = TabExpansion2 -inputScript 'Get-ChildItem | Select-Object @{ ' -cursorColumn 'Get-ChildItem | Select-Object @{ '.Length
         $res.CompletionMatches.Count | Should Be 2
         $res.CompletionMatches.Foreach{$_.CompletionText -in 'Name', 'Expression' | Should Be $true}
-    }        
+    }
 
     It 'Should complete Sort-Object  hashtable' {
-        $res = TabExpansion2 -inputScript 'Get-ChildItem | Sort-Object @{ ' -cursorColumn 'Get-ChildItem | Sort-Object @{ '.Length 
+        $res = TabExpansion2 -inputScript 'Get-ChildItem | Sort-Object @{ ' -cursorColumn 'Get-ChildItem | Sort-Object @{ '.Length
         $res.CompletionMatches.Count | Should Be 3
         $res.CompletionMatches.Foreach{$_.CompletionText -in 'Expression', 'Ascending', 'Descending' | Should Be $true}
-    }    
+    }
 
     It 'Should complete New-Object  hashtable' {
         class X {
             $A
             $B
             $C
         }
-        $res = TabExpansion2 -inputScript 'New-Object -TypeName X -Property @{ ' -cursorColumn 'New-Object -TypeName X -Property @{ '.Length 
+        $res = TabExpansion2 -inputScript 'New-Object -TypeName X -Property @{ ' -cursorColumn 'New-Object -TypeName X -Property @{ '.Length
         $res.CompletionMatches.Count | Should Be 3
         $res.CompletionMatches.Foreach{$_.CompletionText -in 'A', 'B', 'C' | Should Be $true}
-    }     
+    }
+
+    It 'Should complete "Get-Process -Id " with Id and name in tooltip' {
+        $cmd = 'Get-Process -Id '
+        $res = TabExpansion2 -inputScript $cmd  -cursorColumn $cmd.Length
+        $res.CompletionMatches[0].CompletionText -match '^\d+$' | Should be true
+        $res.CompletionMatches[0].ToolTip -match '^\w' | Should be true
```

Comment:
> \w stands for "word character", usually [A-Za-z0-9_]

So `ToolTip -match '^\w'` doesn't guarantee it's in our expected form "<digits - string>".